src/Microsoft.AspNetCore.Mvc.Razor/RazorPage.cs

```diff
@@ -764,8 +763,7 @@ public bool IsSectionDefined(string name)
         /// In layout pages, renders the content of the section named <paramref name="name"/>.
         /// </summary>
         /// <param name="name">The name of the section to render.</param>
-        /// <returns>Returns <see cref="HtmlString.Empty"/> to allow the <see cref="Write(object)"/> call to
-        /// succeed.</returns>
+        /// <returns>An empty <see cref="IHtmlContent"/>.</returns>
         /// <remarks>The method writes to the <see cref="Output"/> and the value returned is a token
         /// value that allows the Write (produced due to @RenderSection(..)) to succeed. However the
         /// value does not represent the rendered content.</remarks>
```

Comment:
Should I remove the 4 or 5 `<remarks>` as well? These say about the same thing as the `<returns>` "to allow" phrases but in gorier detail.